src/runtime/disco/builtin.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#ifndef TVM_RUNTIME_DISCO_BUILTIN_H_
+#define TVM_RUNTIME_DISCO_BUILTIN_H_
+
+#include <tvm/runtime/data_type.h>
+#include <tvm/runtime/module.h>
+#include <tvm/runtime/ndarray.h>
+
+#include "./utils.h"
+
+namespace tvm {
+namespace runtime {
+
+/*!
+ * \brief Load a runtime Module, then create and initialize a RelaxVM
+ * \param path The path to the runtime Module (a DSO file) to be loaded
+ * \param device The default device used to initialize the RelaxVM
+ * \return The RelaxVM as a runtime Module
+ */
+Module LoadVMModule(std::string path, Device device);
```

Comment:
These are all remote functions and likely should not be part of .h file. instead, only register them locally at worker and expose controller/session APIs

Reply:
I exposed them to be used by other worker-side methods. How about keeping them in a `builtin/` or `worker_builtin/` folder?

Reply:
worker_builtin might be ok, although ideally worker side builtin should be minimized, which means they can be contained as a single cc file. The main thing is to avoid being confused from controller side code